examples/client_echo/client_echo.js

```diff
@@ -1,6 +1,6 @@
 const mc = require('minecraft-protocol')
 
-if (process.argv.length !== 4) {
```

Comment:
This is incorrect. Should be not below 3 and not more than 5 like before.
My comment on this on the previous PR was for the example you had added.
(And that you removed apparently ?)
Can you fix this please ?

Reply:
Whoops. I misinterpreted your comment previously. I will solve this.